pom.xml

```diff
@@ -22,8 +22,29 @@
         <ch-kafka.version>1.4.4</ch-kafka.version>
         <kafka-models.version>1.0.12</kafka-models.version>
         <structured-logging.version>1.9.3</structured-logging.version>
+        
+        <maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
+        <maven-shade-plugin.version>3.2.4</maven-shade-plugin.version>
+        <maven-compiler-plugin.version>3.8.1</maven-compiler-plugin.version>
+        <jacoco-maven-plugin.version>0.8.5</jacoco-maven-plugin.version>
```

Comment:
This can go as well If you are happy with the version in parent pom which is 0.8.4

Reply:
I believe this had to remain after using the emergency auth code web project as a base which was the suggestion i was given. see here: https://github.com/companieshouse/emergency-auth-code-web/blob/master/pom.xml

I'm not sure if it's necessary but it may be needed as other projects have included these version declarations